data-raw/assemble_bundled_data.R

```diff
@@ -80,7 +80,7 @@ bl_data$ssm_to_bundle <- read_xlsx(
         "Tumor_Sample_Barcode" = "tumor_biospecimen_id",
         "Matched_Norm_Sample_Barcode" = "normal_biospecimen_id"
     ) %>%
-    select(names(GAMBLR:::maf_header[1:45])) %>%
+    select(names(GAMBLR.helpers:::maf_header[1:45])) %>%
```

Comment:
So now GAMBLR.data depends on GAMBLR.helpers. But GAMBLR.helpers already depends on GAMBLR.data, so we have a circular dependency.

Reply:
This is not a function, or executable stand-alone script. This is just documentation on how the bundled data is generated, so whenever we need add/modify something it can be done reproducibly. This file also depends on GAMBLR.results, GAMBLR.utils, refers to hard-coded path to directories etc. The only reason it is referred to explicitly here is because the maf_header is not exported (again, separate issue to be tackled independently). So, this does not creates circular dependency since has no effect on DESCRIPTION/NAMESPACE.